test/e2e/common/node/container_probe.go

```diff
@@ -113,7 +113,7 @@ var _ = SIGDescribe("Probing container", func() {
 				return false, err
 			}
 			return podutil.IsPodReady(p), nil
-		}, 1*time.Minute, 1*time.Second).ShouldNot(gomega.BeTrue(), "pod should not be ready")
+		}, 1*time.Minute, 1*time.Second).ShouldNot(gomega.BeTrueBecause("pod should not be ready"))
```

Comment:
None of these tests which return a boolean and checking it are done well: if they fail, there is no indication why they failed and what the actual state is.

Avoiding boolean checks would have been a chance to clean that up, but that's probably asking too much, so let's keep this as-is.